src/avfoundation.cs

```diff
@@ -741,6 +933,42 @@ interface AVAudioEngine {
 		[Notification]
 		[Field ("AVAudioEngineConfigurationChangeNotification")]
 		NSString ConfigurationChangeNotification { get; }
+
+		[TV (11, 0), NoWatch, Mac (10, 13), iOS (11, 0)]
+		[Export ("autoShutdownEnabled")]
+		bool IsAutoShutdownEnabled { [Bind ("isAutoShutdownEnabled")] get; set; }
```

Comment:
`AutoShutdownEnabled`
The `Is*` can make sense when no setter is provided - an even then we can't be sure Apple don't add one later